.travis.yml

```diff
@@ -0,0 +1,9 @@
+script:
+- |
```

Comment:
Travis's `script` section can actually be written as 
```
- docker build --target test --tag todo-app:test .
- docker run todo-app:test tests/test_view_model.py
- etc
```

But super minor. Just breaks it up a little in the logs so if something fails it's potentially easier to see what.

Note that Travis's "deploy" script that you may or may not use in module 8 does *not* support a list of commands.